### ISSUE TITLE
 [securedrop-export] Add option to keep a device unlocked on sd-devices

### ISSUE THREAD

**Issue by sssoleileraaa** [freedomofpress/securedrop-client#1729]
Follow-up for https://github.com/freedomofpress/securedrop-export/pull/39

> We will need to create a followup issue with a new status code to inform the client that the device is already unlocked and no user passphrase is necessary.

The issue for this was created here: https://github.com/freedomofpress/securedrop-client/issues/1734

> **Important: ** If the device is unlocked by an external program we will still lock it after a file transfer. This is to err on the side of caution.

No issue was created for this at the time because it was something I thought we wouldn't want to support. But once freedomofpress/securedrop-client#1734 is done, we can revisit this topic and potentially add a new option to keep the device unlocked while continuing to default to locking the device after a transfer completes. If we did this, I think it would make sense to say something more like "Keep device unlocked" rather than "Remember password." 

## User story

As a journalist who wants to export multiple documents, I'd like to keep the luks-encrypted thumb drive unlocked until I finish all my exports.

**Comment by rocodes** [comment]
Now that #1777 has been merged, this change will be straightforward to implement, but still needing some UX consideration:

- In one (or more) ExportWizardPage, add a QCheckbox that defaults to checked (true) that says "lock this device after export" or similar. use `registerField` the same way the passphrase is a registered field, so that the wizard has access to the boolean value of this checkbox
- Pass the result to sd-devices, either via a json metadata field, if we are still using json metadata, or via a qrexec service argument, if we have transitioned to that already.
- There is already logic in `export/disk/cli.py` that unmounts and closes the volume only if `should_close_volume` is True (True by default, so currently it always closes and locks the volume).

The UX question to consider is where this option should be presented. The only page that users are _guaranteed_ to see is the Preflight page. Everything else is conditional on what state their device is in. The easiest option is probably to add a "Review before Exporting" page that offers this checkbox, along with a summary of what is about to be exported.

**Comment by eloquence** [comment]
This is still relevant for the rewrite, though we may want to close and open it with a clearer scope. Keeping open and milestoning this one for now.
